src/object/thunderstorm.cpp

```diff
@@ -141,6 +153,10 @@ Thunderstorm::lightning()
   if (!m_strike_script.empty()) {
 	  Sector::get().run_script(m_strike_script, "strike-script");
   }
+
+  change_background_colors(true);
+  Sector::current()->get_singleton_by_type<AmbientLight>().set_ambient_light(Color::LIGHTNING_HIT_COLOR);
+  layer = std::numeric_limits<int>::max();
```

Comment:
There's an enum for layers. It's in layer.hpp. You should use, use LAYER_LIGHTMAP.